src/App.tsx

```diff
@@ -1,40 +1,55 @@
-import React from 'react';
+import { useState } from 'react';
+import type { FC } from 'react';
+import { Tabs } from './components/Tabs';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-export const tabs = [
+export interface Tab {
+  id: string;
+  title: string;
+  content: string;
+}
+
+export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App: React.FC = () => {
+export const App: FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0].id);
+
+  const selectTab = (tabId: string) => {
+    if (tabs.find((tab) => tab.id === tabId)) {
+      setSelectedTab(tabId);
+    } else {
+      setSelectedTab(tabs[0].id);
+    }
+  };
```

Comment:
I'd write it this way
```suggestion
  const selectTab = (tabId: string) => {
    const isSelectedTabIdValid = tabs.find((tab) => tab.id === tabId;
    const tabToSelect = isSelectedTabIdValid
      ? tabId
      : tabs[0].id
      setSelectedTab(tabId);
  
    setSelectedTab(tabToSelect)
  };
```